project/frontend/src/pages/MatchResults.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+class MatchResults extends React.Component {
+  render() {
+    const { data } = this.props.location;
```

Comment:
is it possible to name this restaurantsList or something more descriptive than data?